docs/issues_catched.md

```diff
@@ -0,0 +1,30 @@
+
+# Issues Precaution finds
+
+Precaution utilizes multiple security static code analysis tools (linters)
+to identify issues for a particular language.
+Therefore the amount and types of checks for the different programming languages differ.
+
+However, some of the rules used by the different linters can be grouped:
+
```

Comment:
This introduction feels a bit awkward, how about:

> Precaution uses language-specific security static code analysis tools (linters) to identify issues, therefore the amount and types of checks vary by programming language. Some of the classes of issue which Precaution detects include:

Reply:
Sounds a lot cleaner and easier to read.